draft-irtf-cfrg-frost.md

```diff
@@ -452,9 +455,9 @@ This section describes the subroutine for creating the per-message challenge.
     return challenge
 ~~~
 
-# Two-Round FROST {#frost-spec}
+# Two-Round FROST Signing Protocol {#frost-spec}
 
-FROST is a two-round threshold signature protocol for producing Schnorr signatures.
+We now present the two-round variant of the FROST threshold signature protocol for producing Schnorr signatures.
```

Comment:
Nit: FROST (flexible round-optimized schnorr-based threshold) threshold signature protocol?

maybe: 
```suggestion
We now present the two-round variant of the FROST signature protocol for producing Schnorr signatures in the threshold setting.
```

Reply:
FROST is a threshold signature protocol, so going with the original language.